powerquery-docs/connectors/clickhouse.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Power Query ClickHouse connector
+description: Provides basic information, prerequisites, and instructions on how to connect to ClickHouse
+author: bentsileviav
+#ms.topic: conceptual
+#ms.date: 8/11/2024
+#ms.author: bentsileviav
+---
+
+# ClickHouse
+
+> [!NOTE]
+> The following connector article is provided by ClickHouse, Inc. the owner of this connector and a member of the
+> Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or
+> have changes you would like to see made to this article, visit the ClickHouse website and use the support channels
+> there.
+
+## Summary
+
+| Item                           | Description                                                                    |
+|--------------------------------|--------------------------------------------------------------------------------|
+| Release State                  | Preview                                                                        |
+| Products                       | Power BI (Semantic models)<br/>Power BI (Dataflows)<br/>Fabric (Dataflow Gen2) |
+| Authentication Types Supported | ClickHouse (Username/Password)                                                 |
+
+> [!NOTE]
+> Some capabilities may be present in one product but not others due to deployment schedules and host-specific
+> capabilities.
+
+## Prerequisites
+
+* A running ClickHouse setup
+* Customers must install the [ClickHouse ODBC driver](https://github.com/ClickHouse/clickhouse-odbc) before using the
+  connector
```

Comment:
Do we need a link to the docs on the official website? https://clickhouse.com/docs/en/integrations/powerbi For example, the doc doesn't provide details on where to get user credentials on ClickHouse Cloud

Reply:
Added "More Resources" section.